wherescape/connectors/hubspot/python_hubspot_check_missing_columns.py

```diff
@@ -26,7 +27,7 @@ def compare_columns(wherescape_columns, hubspot_columns):
 def create_table_rows(
     missing_in_wherescape, missing_in_hubspot, environment, table_name
 ):
-    # Creating the table row list for the insert query
+    """Function that creates the table row list for the insert query"""
```

Comment:
[PEP 257](https://peps.python.org/pep-0257/) recommends that the first line of a docstring is written in the form of a command, ending in a period.